pyproject.toml

```diff
@@ -64,7 +64,7 @@ omit = [
 name = "ehrql"
 
 [tool.black]
-extend-exclude = "(tests/acceptance/external_studies/.*|tests/fixtures/bad_dataset_definitions/.*)"
+extend-exclude = "(tests/acceptance/external_studies/.*|tests/fixtures/bad_dataset_definitions/.*|docs/)"
```

Comment:
comment: `extend-exclude` does *not* apply when running `black` directly on a filename; that's `force-exclude`. This can result in editors/IDEs that work by directly passing the filename to `black` still causing an unintended reformat.

If we really want to avoid a reformatting, perhaps we should be using `force-exclude` which applies in all circumstances? (You can [override the exclusion](https://stackoverflow.com/a/73296261) by `black --force-exclude='' path/to/your-excluded-file.py` at the command line.

(This is not directly related to this PR, but did arise while working on it.)